tests/custom.rs

```diff
@@ -6,45 +6,10 @@
     not(feature = "js")
 ))]
 
-use wasm_bindgen_test::wasm_bindgen_test as test;
 #[cfg(feature = "test-in-browser")]
 wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
We can delete this if we are importing the `common` module.

Reply:
> We can delete this if we are importing the common module.

Good catch. I pushed an updated version with this change.